pkg/istio-agent/xds_proxy.go

```diff
@@ -153,7 +191,7 @@ func initXdsProxy(ia *Agent) (*XdsProxy, error) {
 			req = &discovery.DiscoveryRequest{
 				TypeUrl: v3.HealthInfoType,
 				ErrorDetail: &google_rpc.Status{
-					Code:    500,
+					Code:    13,
```

Comment:
What is error code 13 ?

Reply:
@howardjohn - this was part of your initial commit. No idea what error code 13 is.

Reply:
https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto#L169

We should use a constant though, I am sure one exists in google_rpc library or similar. can revert it here if you want

Reply:
Added the relevent grpc error code